### PR TITLE
sys-process/rtirq: Initial commit.

### PR DESCRIPTION
**Pull request by lilrc**
Package-Manager: portage-2.3.0_rc1

### CODE REVIEW

File: sys-process/rtirq/rtirq-20150216.ebuild [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+src_install(){
+	dosbin ${PN}.sh
+	doinitd "${FILESDIR}"/${PN}
+	newconfd ${PN}.conf ${PN}
+}
+
+pkg_postinst(){
+	if [ "$(rc-config list default | grep rtirq)" = "" ] ; then
```
Comment (monsieurp) [inline sys-process/rtirq/rtirq-20150216.ebuild]: Hmm this is sh syntax AFAIK. Let's be a bit more pedantic and switch to the bash syntax with double [[ ]]'s.

File: sys-process/rtirq/rtirq-20150216.ebuild [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		elog "You probably want to add rtirq to the default runlevel, i.e."
+		elog "  rc-update add rtirq default"
+	fi
+	if [ "$(uname -r | grep rt)" = "" ] ; then
```
Comment (monsieurp) [inline sys-process/rtirq/rtirq-20150216.ebuild]: Ditto

**Comment by monsieurp** [comment]
Hi and thanks for the PR. Can you give us a bit more background as to the usefulness/utility of this package? Has someone filed a bug report for it already?

**Comment by lilrc** [comment]
Okay, the brackets are fixed.

This package has up until now been maintained in the proaudio overlay, but there is not much happening in the overlay and the current ebuild there does not fetch (upstream removes old versions as soon as new are released), so I have decided to maintain the packages of proaudio I need in gx86.

A crucial part of professional realtime audio is low latency. rtirq sets realtime priorities and scheduling policies for desired IRQ threads on boot. Without rtirq you would need to set these priorities yourself after each boot or live with sound dropouts. See also [1].

Related gentoo bug: 566678
[1] http://alsa.opensrc.org/Rtirq

**Comment by lilrc** [comment]
Thank you!

**Pull request closed without merging**